File: app/src/main/java/com/example/sunnygym/QRCodeActivity.java
package com.example.sunnygym;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class QRCodeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_qrcode);

        Button backToProfileButton = findViewById(R.id.backToProfileButton);

        // Botón para regresar al perfil
        backToProfileButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(QRCodeActivity.this, UserProfileActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: APP_Kotlin/app/src/main/java/com/example/sunnygym/QRCodeActivity.kt
package com.example.sunnygym

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class QRCodeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_qrcode)

        val backToProfileButton = findViewById<Button>(R.id.backToProfileButton)

        // Botón para regresar al perfil
        backToProfileButton.setOnClickListener {
            val intent = Intent(this@QRCodeActivity, UserProfileActivity::class.java)
            startActivity(intent)
        }
    }
}